policies/invited-expert.md

```diff
@@ -0,0 +1,37 @@
+## Invited Expert Policy for the W3C Web of Things (WoT) Working Group
+
+### W3C Policy
+
+This document extends and clarifies the W3C Invited Expert Policy at https://www.w3.org/invited-experts/ for use by the WoT Working Group and WoT Interest Group.
+
+Some items from the W3C document are repeated for convenience.
+
+### By Invitation
+
+Invited Experts join the Working Group by invitation from the Working Group Chairs and Team Contact. 
+
+Proposals, requests, and suggestions may be made to the Chairs for Invited Experts, and the Chairs may consider extending invitations. 
+
+If a proposal is submitted to the Chairs for consideration, the Chairs will discuss with the Team Cointact and decide whether to extend an invitation.
+
+### Requirements
+
+To be considered as an Invited Expert in the WoT Working Group, an invitee must make a commitment to contribute to the Web of Things specifications and must demonstrate the ability to contribute to the specifications. 
+
+An Invitee must prepare and present a brief - 10 minutes - summary presentation of their background and experience, particularly in standards organizations, ans explain what/how they plan to contribute to the Web of Things specification.
+
+An Invited Expert must be a contributing member of the group and is expected to regularly join WoT meetings, perform administrative duties, make and review Merge Requests, etc.
+
+### Process
+
+THe Chairs and Team contact may from time to time extend an invitation to someone to join the WOT Working Group or Interest Group as an Invited Expert. Often there will be a suggestion or request sent to the Chairs and Team Contact to join the Web of Things as an Invited Expert.
+
+Consideration will be given to invitees during the weekly WoT Chairs call. The Chairs will discuss each potential invitee and typically decide whether to extend an invitation within 2 or 3 weeks.
+
+On completing the decision process, the Chairs will notify the potential invitee or sponsors in a timely manner.
```

Comment:
sponsor is not previously defined

Reply:
Removed the reference to sponsors: 
...Chairs will notify the potential invitee in a timely manner.